linux_pipeline/Jenkinsfile_kernel_bisect_commit_validator

```diff
@@ -56,17 +56,24 @@ properties ([
             choices: 'read\nrandread\nwrite\nrandwrite',
             name: 'PERF_IO_MODE',
             description: '[FIO] IO_MODE'],
+        [$class: 'ChoiceParameterDefinition',
+            choices: 'KVP\nCore',
+            name: 'FUNC_TEST_SUITE',
+            description: '[FUNC] the test suite which you want to run.'],
+        [$class: 'StringParameterDefinition',
+            name: 'FUNC_XML_PATH',
+            defaultValue: '',
+            description: '[FUNC] path to custom xml'],
         [$class: 'StringParameterDefinition',
             name: 'ENABLED_STAGES',
-            defaultValue: 'build_artifacts, boot_test, perf_hyperv',
+            defaultValue: 'build_artifacts, boot_test, perf_hyperv, func',
```

Comment:
for enabled stages, use the validation_functional_hyperv and validation_performance_hyperv, like in the other pipelines